src/AccessibilityInsights.SharedUx/Controls/SettingsTabs/ApplicationSettingsControl.xaml.cs

```diff
@@ -204,7 +204,7 @@ public bool IsConfigurationChanged(ConfigurationModel config)
             {
                 return true;
             }
-            if (config.PlayScanningSound != (this.DataContextVM.PlayScanningSound))
+            if ((int) config.SoundFeedback != (this.DataContextVM.SoundFeedbackMode))
```

Comment:
For consistency with the HighligherMode code, maybe the cast should be on the right hand side?
```suggestion
            if ((config.SoundFeedback != (SoundFeedbackMode)(this.DataContextVM.SoundFeedbackMode))
```